src/altinn-app-frontend/src/features/form/components/__snapshots__/MessageBanner.test.tsx.snap

```diff
@@ -0,0 +1,15 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`form/components/MessageBanner.tsx form/components/MessageBanner -- should match snapshot 1`] = `
```

Comment:
Don't know if we have taken any real decision here - but the sentiment in the team when you have been gone is that snapshot tests don't really offer us any clear value and have not really been added to newer tests. Do you have any other thoughts on this?

Reply:
I do agree actually, and that is also the reason why I didn't do a snapshot to test the red background. Guess I just followed the setup of the old tests blindly 🙈  I'll update the default test case to also just test more specifically for what we would like to see, rather than using a snapshot.